src/container/cli.c

```diff
@@ -1868,15 +1869,22 @@ cont_req_cleanup(enum creq_cleanup_stage stage, struct cont_req_arg *args)
 	switch (stage) {
 	case CLEANUP_ALL:
 		crt_req_decref(args->cra_rpc);
+		break;
 	case CLEANUP_BULK:
 		if (args->cra_bulk)
 			crt_bulk_free(args->cra_bulk);
+		break;
 	case CLEANUP_RPC:
 		crt_req_decref(args->cra_rpc);
+		break;
 	case CLEANUP_POOL:
 		dc_pool_put(args->cra_pool);
+		break;
 	case CLEANUP_CONT:
 		dc_cont_put(args->cra_cont);
+		break;
+	default:
+		break;
```

Comment:
seems like the original code is intended, better to confirm with @liw

Reply:
Woops, I this it is intended after encountered few errors in CI unit test and core test. Reverted pool/cli.c, but missed container/cli.c, will revert this.